inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.cpp

```diff
@@ -80,9 +88,94 @@ void MKLDNNPadNode::initSupportedPrimitiveDescriptors() {
     config.outConfs[0].constant = false;
 
     auto memoryFormat = MKLDNNMemory::GetPlainFormat(getParentEdgeAt(0)->getDims());
-    config.inConfs[0].desc = MKLDNNMemoryDesc(getParentEdgeAt(0)->getDims(), dataType, memoryFormat);
-    config.outConfs[0].desc = MKLDNNMemoryDesc(getChildEdgeAt(0)->getDims(), dataType, memoryFormat);
-    supportedPrimitiveDescriptors.push_back({config, impl_desc_type::unknown, memoryFormat});
+    auto pushSupportedPrimitiveDescriptor = [&](memory::format memoryFormat) {
+        config.inConfs[0].desc = MKLDNNMemoryDesc(getParentEdgeAt(0)->getDims(), dataType, memoryFormat);
+        config.outConfs[0].desc = MKLDNNMemoryDesc(getChildEdgeAt(0)->getDims(), dataType, memoryFormat);
+        supportedPrimitiveDescriptors.push_back({config, impl_desc_type::ref, memoryFormat});
+    };
+    pushSupportedPrimitiveDescriptor(memoryFormat);
```

Comment:
Do we need explicit `memoryFormat` variable? Lets write stm like: pushSupportedPrimitiveDescriptor(MKLDNNMemory::GetPlainFormat(getParentEdgeAt(0)->getDims()));